edenai_apis/apis/mindee/mindee_api.py

```
@@ -1,9 +1,12 @@
 import json
 from collections import defaultdict
+from io import BufferedReader
+from typing import Dict, Sequence, TypedDict
+import aiofiles
+
 from io import BufferedReader, BytesIO
```

Ruff (0.14.5)

Redefinition of unused `BufferedReader` from line 3

Remove definition: `BufferedReader`

(F811)